docs/Team.adoc

```diff
@@ -168,18 +168,37 @@ Area Lead for: UI [2016 May - 2017 May]
 
 '''''
 
-== Contributors
+=== Contributors
 
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Aokkhoy[Akshay Narayan]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Abrandonyeoxg[Brandon Yeo]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Aweikangchia[Chia Wei Kang]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Acheec[Clarence Chee]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3A500poundbear[Derek]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3AKnewYouWereTrouble[Desmond Ang]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3ANeurrone[Dickson Tan]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Aedmundmok[Edmund Mok]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Aeugenepeh[Eugene Peh]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Amadsonic[Gerald Ng]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Adonjar[Herbert Ilhan Tanujaya]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Ajeffryhartanto[Jeffry Hartanto]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Ajoanneong[Joanne Ong]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Ajaeoheeail[Joel Foo]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Aluyangkenneth[Kenneth Lu]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Adezhanglee[Lee De Zhang]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Ajia1[Lee Jia Yee]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3ALiXiaoooowei[Li Xiao Wei]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3ACrsytral[Lim Shun Xian]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Alouietyj[Louie Tan]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Anicholaschuayunzhi[Nicholas Chua]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Amauris[Sam Yong]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Atshradheya[Shradheya Thakre]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Azzzzwen[Song Zhiwen]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Athenaesh[Thenaesh Elango]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Alinhany[Yong Lin Han]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3AZhangYiJiang[Zhang Yi Jiang]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3AVeryLazyBoy[Zi Yang]
+
+== Current Team (non-addressbook)
```

Comment:
> I put the Current Team (non-addressbook) segment after the Current Team (addressbook) instead, because it seems more intuitive to put the major segment before the minor segment.

Hmm, okay, then perhaps this can be `(Others)` rather than `(non-addressbook)` then (I previously used "non-addressbook" instead of "Others" because of the ordering, but since we move it down here, I think can just use "Others").